src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,48 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.models.Manufacturer;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao =
+                (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
+        Manufacturer manufacturer = new Manufacturer();
+        // getAll data from DataBase
+        List<Manufacturer> manufacturerList = manufacturerDao.getAll();
+        for (Manufacturer m : manufacturerList) {
+            System.out.println(m);
+        }
+        // initialize field values using setters or constructor
+        manufacturer.setName("TestManufacturer");
+        manufacturer.setCountry("TestCountry");
+        manufacturerDao.create(manufacturer);
+        // test other methods from ManufacturerDao
+        // get by id
+        System.out.println(manufacturerDao.get(2L));
+        System.out.println(manufacturerDao.get(20L));
```

Comment:
I dont understand why we use here 2L and 20L... Lets implement more universal solution
```suggestion
        System.out.println(manufacturerDao.get(manufacturer.getId()));
```